sdk/core/azure-core-test/src/test_proxy_manager.cpp

```diff
@@ -225,12 +224,50 @@ void TestProxyManager::SetProxySanitizer()
   Azure::Core::Url matcherRequest(m_proxy);
   matcherRequest.AppendPath("Admin");
   matcherRequest.AppendPath("SetMatcher");
-  const std::string matcherBody
-      = "{\"compareBodies\": false ,\"ignoreQueryOrdering\": true , \"ignoredHeaders\": "
-        "\"x-ms-copy-source,x-ms-proposed-lease-id,x-ms-lease-id,x-ms-file-change-"
-        "time,x-ms-file-creation-time,x-ms-file-last-write-time,x-ms-destination-"
-        "lease-id,x-ms-source-lease-id,x-ms-source-content-crc64,x-ms-content-crc64,x-ms-source-"
-        "content-md5,x-ms-content-md5,Content-MD5\"}";
+  std::string matcherBody;
+  {
+    auto jsonRoot = Json::_internal::json::object();
+    jsonRoot["compareBodies"] = false;
+    jsonRoot["ignoreQueryOrdering"] = true;
+    const std::vector<std::string> excludedHeaders = {
+        "Expect",
+        "Connection",
+    };
+    jsonRoot["excludedHeaders"] = std::accumulate(
```

Comment:
Why use std::vector and std::accumulate instead of using initializer syntax?

In other words, something like:
```c++
jsonRoot["excludedHeaders"] = { "Expect", "Connection" };
```

Reply:
@LarryOsterman Your code will emit string:

```
{"excludedHeaders":["Expect","Connection"]}
```

What we want is:

```
{"excludedHeaders": "Expect,Connection"}
```

This is defined by the test-proxy, we have to follow this grammer.

Reply:
@#$@#$ - in other words, test-proxy's schema is broken JSON (there's no way that parameter should be a string instead of an array). Sigh. 

So be it.